<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addMonster">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeMonster">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QListWidget" name="monsterList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>256</width>
      <height>291</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Not a goblin</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="combatList">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>256</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CR Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="crTotal">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="dbTestLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Workiong</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDbButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Db</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
